docs/docs/09-troubleshooting.md

```diff
@@ -1,28 +1,5 @@
 ## Troubleshooting
 
-### Node built-in could not be resolved
-
-```
-✖ /my-application/node_modules/dep/index.js
-  "http" (Node.js built-in) could not be resolved.
-```
-
-Some packages are written with dependencies on Node.js built-in modules. This is a problem on the web, since Node.js built-in modules don't exist in the browser. For example, `import 'path'` will run just fine in Node.js but would fail in the browser.
-
-**To solve this issue:** you can either replace the offending package ([pika.dev](https://pika.dev/) is a great resource for web-friendly packages) or add Node.js polyfill support:
-
-```js
-// snowpack.config.js
-// Plugin: https://github.com/ionic-team/rollup-plugin-node-polyfills
-module.exports = {
-  installOptions: {
-    rollup: {
-      plugins: [require('rollup-plugin-node-polyfills')()],
-    },
-  },
-};
-```
-
```

Comment:
I understand slimming this section down, but I think having that exact error message on this page still is super helpful. WDYT about leaving this section, and the error message, but linking to the heading above? That way people googling that error can find the appropriate section without having to read everything.

Reply:
The error message here is actually outdated, the new one reads: `Import "${id}" (Node.js built-in) is not available in the browser. Run with --polyfill-node to fix.` 

Given how concise the error is now, it feels more useful to send users directly to this section of the docs.